runtime/compiler/control/J9Options.cpp

```diff
@@ -1229,6 +1229,25 @@ static bool JITServerParseCommonOptions(J9JavaVM *vm, TR::CompilationInfo *compI
 
    return true;
    }
+
+static void JITServerParseLocalSyncCompiles(J9JavaVM *vm, TR::CompilationInfo *compInfo, bool isFSDEnabled)
+   {
+   const char *xxJITServerLocalSyncCompilesOption = "-XX:+JITServerLocalSyncCompiles";
+   const char *xxDisableJITServerLocalSyncCompilesOption = "-XX:-JITServerLocalSyncCompiles";
+
+   int32_t xxJITServerLocalSyncCompilesArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxJITServerLocalSyncCompilesOption, 0);
+   int32_t xxDisableJITServerLocalSyncCompilesArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxDisableJITServerLocalSyncCompilesOption, 0);
+
+   // We either obey the command line option, or make sure to disable LocalSyncCompiles if
+   // something is set that interferes with remote async recompilations.
+   if ((xxDisableJITServerLocalSyncCompilesArgIndex > xxJITServerLocalSyncCompilesArgIndex) ||
+       ((xxJITServerLocalSyncCompilesArgIndex <= 0) &&
```

Comment:
Just wanted to make sure that, `xxJITServerLocalSyncCompilesArgIndex` cannot be 0 when command line option contains `-XX:+JITServerLocalSyncCompiles` as its first option.

Reply:
I think it might be possible. I was misremembering other bits of the code - it does seem to use `>= 0` to test if the option is set.

Reply:
Please double-check

Reply:
The value of the index appears to be 9 if it's the first option

Reply:
In any case, I think the argument-parsing function returns -1 if the option isn't found, so the test could equally well be `== -1`